src/components/portfolio/DetailsAnalyticsDebtEquityBar.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Chart from 'react-apexcharts';
+import { useTranslation } from 'react-i18next';
+import { useTheme } from '@material-ui/core';
+import { NonEmptyPortfolioDetails } from '../../portfolio/APIClient';
+
+type DetailsAnalyticsDebtEquityBarProps = {
+  portfolio: NonEmptyPortfolioDetails;
+  height: number;
+};
+
+const DetailsAnalyticsDebtEquityBar: React.FC<DetailsAnalyticsDebtEquityBarProps> = ({
+  portfolio,
+  height,
+}) => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const biggestTen = portfolio.positions
+    .sort(
+      (pos1, pos2) => pos2.qty * pos2.stock.price - pos1.qty * pos1.stock.price
+    )
+    .slice(0, 9);
```

Comment:
The default order of sort() is ascending, this would take the 10 smallest. (Don't know if this is the best way to do this but it avoids calling .reverse())
```suggestion
  const biggestTen = portfolio.positions
    .sort(
      (pos1, pos2) => pos2.qty * pos2.stock.price - pos1.qty * pos1.stock.price
    )
    .slice(positions.length > 10 ? positions.length - 10 : 0);
```

Reply:
If pos2 is of greater value than pos1 my expression after => evaluates to something grater than 0. In that case pos2 will be put in front of pos1. In other words the largest values come first in the array. In the end I take the first ten values of the array. Why is that wrong?